librz/util/list.c

```diff
@@ -126,7 +126,7 @@ RZ_API void rz_list_purge(RZ_NONNULL RzList *list) {
  * \brief Empties the list and frees the list pointer
  *
  **/
-RZ_API void rz_list_free(RZ_NONNULL RzList *list) {
+RZ_API void rz_list_free(RzList *list) {
 	if (list) {
```

Comment:
Maybe flip too?

Reply:
it's just 2 instructions, that's why i didn't change it. I think it can be changed if we ever need to change rzlist(though i doubt).